debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/OutboxEventRouterIT.java

```diff
@@ -205,5 +228,126 @@ public void shouldSupportAllFeatures() throws Exception {
         assertThat(valueStruct.getString("aggregateType")).isEqualTo("UserEmail");
         assertThat(valueStruct.getInt32("eventVersion")).isEqualTo(1);
         assertThat(valueStruct.getBoolean("someBoolType")).isEqualTo(true);
+        assertThat(valueStruct.getBoolean("deleted")).isEqualTo(false);
+    }
+
+    @Test
+    public void shouldConsumeDeleteEvents() throws Exception {
```

Comment:
Maybe this test is from the previous implementation? I have the feeling it's not actually testing anything with the new feature

Reply:
Yeah this was from the previous behavior I had implemented. I should rework this test.

Reply:
Right; I've repurposed this method now to test the case that no tombstone is emitted for null if that option isn't enabled.